Latex-Briefvorlage.tex

```diff
@@ -40,6 +40,10 @@
 
 % Zeichenkodierung des Dokuments ist in UTF-8
 \usepackage[utf8]{inputenc}
+% Schriftart ist Latin Modern
+\usepackage{lmodern}
```

Comment:
Ich sehe mit dieser Anweisung in der Schrift keinen Unterschied.

Reply:
Siehe http://namsu.de/Extra/pakete/Lmodern.html

Reply:
Bist auf die Positionierung der Anführungszeichen und I-Punkte sehe ich immer noch keinen Unterschied. Kann es sein, dass diese Unterschiede früher stärker waren?